packages/block-editor/src/hooks/use-bindings-attributes.js

```diff
@@ -139,6 +140,11 @@ export const withBlockBindingSupport = createHigherOrderComponent(
 					continue;
 				}
 
+				// Populate context.
+				for ( const key of source.usesContext || [] ) {
```

Comment:
With https://github.com/WordPress/gutenberg/pull/65599/files#r1772886078 applied, this can be refactored to:

```js
let updatedContext = context;
if ( source.usesContext?.length ) {
    let updatedContext = { ...context };
    for ( const key of source.usesContext ) {
        updatedContext[ key ] = blockContext[ key ];
    }
}
```